Application/EdFi.Ods.Common/Configuration/DefaultPageSizeLimitProvider.cs

```diff
@@ -3,21 +3,19 @@
 // The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 // See the LICENSE and NOTICES files in the project root for more information.
 
+#if NETSTANDARD
 using Microsoft.Extensions.Configuration;
 
-#if NETSTANDARD
 namespace EdFi.Ods.Common.Configuration
 {
     public class DefaultPageSizeLimitProvider : IDefaultPageSizeLimitProvider
     {
-        private readonly IConfigValueProvider _configValueProvider;
-
-        public DefaultPageSizeLimitProvider(IConfigValueProvider configValueProvider)
+        private readonly IConfiguration _configuration;
+        public DefaultPageSizeLimitProvider(IConfiguration configuration)
         {
-            _configValueProvider = configValueProvider;
-        }
-
-        public int GetDefaultPageSizeLimit() => int.TryParse(_configValueProvider.GetValue("defaultPageSizeLimit"), out int defaultPageSizeLimit)
+            _configuration = configuration;
+       }
+        public int GetDefaultPageSizeLimit() => int.TryParse(_configuration.GetSection("defaultPageSizeLimit").Value, out int defaultPageSizeLimit)
```

Comment:
formatting issue here. this needs to be fixed please. also fix the braces , and there should be a blank line between the statements

Reply:
Installed resharper and formatted the code